versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
```

Comment:
```suggestion
  # @TODO: we must move to google provider 5.x version to use severity. Actually, we cannot set srverity since zambon-ops is using a 4.x version.
  required_providers {
```